docs/source/modin/supported/index_supported.rst

```diff
@@ -138,7 +139,7 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``notna``                   | N                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``astype``                  | P                               |                                  |                                                    |
+| ``astype``                  | P                               |                                  | WIP: SNOW-1458125                                  |
```

Comment:
Can we avoid referring to Jira tickets here? I'd say make it `N` for now, and the other PR can turn it into `P`.